RELEASE_INFO-6.7.md

```diff
@@ -56,6 +56,9 @@ Payment tokens are no longer deleted immediately after use. A new scheduled task
 ### Added sanitized HTML tag support for app snippets
 Added sanitized HTML tag support for app snippets. App developers can now use HTML tags for better formatting within their snippets. The sanitizing uses the `basic` set of allowed HTML tags from the `html_sanitizer` config, ensuring that security-related tags such as `script` are automatically removed.
 
+### Improve request limit handling for page listing
+Changed `\Shopware\Core\Content\Product\SalesChannel\Listing\Processor\PagingListingProcessor::getLimit` to improve fallback handling to allow headless requests to use the product listing limit configuration
```

Comment:
Release Infos like this are not needed IMHO, we should only document changes that are really relevant, and for those changes we should document what the consequence of that change is, new patterns / features you need to use or workarounds that are not needed anymore.

Simply describing what you changed is not needed anymore, as that information can be automtically be retrieved from the commit already, so there is no added value